docs/tutorials/hello-tye/00_run_locally.md

```diff
@@ -7,8 +7,7 @@ This tutorial will demonstrate how to use [`tye run`](/docs/reference/commandlin
 1. Make a new folder called `microservice` and navigate to it:
 
     ```text
-    mkdir microservice
-    cd microservice
+    take microservice
```

Comment:
I haven't heard of this command before, and I don't see it available on both ubuntu or powershell. 

The commands above are generally available on most shells, so I'd prefer to keep it as is.

Reply:
Agree